core/shared/platform/linux-sgx/sgx_ipfs.c

```diff
@@ -78,6 +83,27 @@ ipfs_file_destroy(void *sgx_file)
     sgx_fclose(sgx_file);
 }
 
+// Writes a given number of zeroes in file at the current offset.
+// The return value is zero if successful; otherwise non-zero.
+static int
+ipfs_write_zeroes(void *sgx_file, size_t len)
+{
+    int min_count;
```

Comment:
Shall we check `len == 0` firstly, or `sgx_fwrite(zeroes_padding, 1, min_count, sgx_file)` will be called with `min_count == 0`?
```C
if (len == 0)
    return 0;
```

Reply:
Hey @wenyongh, thanks for the code review.

I preferred to convert the `do.. while` loop into a `while` loop, which has the same predicate, rather than adding an `if` statement. This should simplify the code and verify that the value of the argument `len` is greater than 0.

Cheers

Reply:
Good, thanks!